tests/utils/calendar-helper.spec.ts

```diff
@@ -2,6 +2,15 @@ import dayjs, { Dayjs } from "dayjs";
 import { CalendarHelper } from "../../src/util/calendar-helper";
 
 describe("CalendarHelper", () => {
+    beforeAll(() => {
+        jest.useFakeTimers();
+        jest.setSystemTime(new Date("26 Oct 2023 12:00:00 GMT").getTime());
```

Comment:
minor thing but a Date instance is also accepted so there's no need to convert it

```suggestion
        jest.setSystemTime(new Date("26 Oct 2023 12:00:00 GMT"));
```